hack/build-local-images.py

```diff
@@ -227,4 +236,4 @@ def debug(message):
     print "[ERROR] This script knows how to build:\n\t{}".format(
         "\n\t".join(map(full_name, image_config.keys()))
     )
-    exit(1)
\ No newline at end of file
+    exit(1)
```

Comment:
Can I have this change?  No trailing newline at the EOF tickles my delicate sensibilities ;-)